maine-thesis.cls

```diff
@@ -79,6 +79,18 @@
 \RequirePackage[none]{hyphenat}[2009/09/02 v2.3c hyphenation utilities]
 \AfterPackage{hyperref}{% Graduate School requirements mandate a slight modification to hyperref's redefinition of the caption command.  This is implemented here.  The command is only executed after hyperref is loaded in the document (if it is loaded).
 \long\def\@caption#1[#2]#3{%
+\hypersetup{ pdfdisplaydoctitle=true}
+
+\AtBeginDocument{
+\hypersetup{	
+	pdftitle=\@title,
+	pdfauthor=\@author,
+	pdfcreator=\relax,
+	pdfproducer=\relax % These could be changed, but for the thesis, I left them blank.
+	}
+}
+\pdfsuppressptexinfo=-1 %to remove some useless tags generated by pdftex
```

Comment:
This command is specific to PDFTeX and not everyone uses that engine (I usually use LuaTeX these days, for instance).  Please add the `iftex` package to package loading and then enclose this line in the `\ifPDFTeX` conditional.